scpdt/plugin.py

```diff
@@ -99,16 +102,16 @@ def collect(self):
         # We plugin scpdt's `DTFinder` that uses the `DTParser` which parses the doctest examples 
         # from the python module or file and filters out stopwords and pseudocode.
         finder = DTFinder(config=dt_config)
-
-        # the rest remains unchanged
         optionflags = doctest.get_optionflags(self)
-        runner = doctest._get_runner(
+
+        # We plugin scpdt's `DebugDTRunner`
+        runner = _get_runner(
             verbose=False,
             optionflags=optionflags,
-            checker=_get_checker(),
-            continue_on_failure=doctest._get_continue_on_failure(self.config),
```

Comment:
why are we dropping continue_on_failure?

Reply:
https://github.com/ev-br/scpdt/pull/87#issuecomment-1677434321 please make sure that this test passes with the plugin